sdk/communication/CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing Guide
+
+This a contributing guide made specifically for the Azure Communication Services SDK. The Azure SDK repo also has a contributing guide that might help you in some other general processes this guide assumes you have done. If you haven't checked that one out yet, you can find it [here](https://github.com/Azure/azure-sdk-for-java/blob/master/CONTRIBUTING.md)
+
+The Azure Communication Services SDK for Java currently consists of 5 different packages. While each package has its own set of environment variables to make their tests run successfully, all of them follow a similar structure that allows a smooth onboarding process.
+
+Let's get started with how to setup the repo itself.
+
+## Installation process
+
+To get started with any of the packages, change directory to the root communication folder and run the `mvn verify -DskipTests` command. This will install all of the local files necessary for you to run the corresponding tests for all packages. 
+
+Once the packages have been installed on your machine, let's jump on how to run the tests to see that everything is in order.
+
+## Testing
+
+Make sure to check out the general contributing guide the Azure SDK repo has for a more in-depth look at testing and setting up your dev environment. You can check out the contributing file [here](https://github.com/Azure/azure-sdk-for-java/blob/master/CONTRIBUTING.md)
+
+
+When you go inside the tests folder of the package you are working with, you will see a folder called `resources` that has another folder named `session-records`. This folder contains, as its name suggests, recordings of successful calls to the API that allow us to run the tests in PLAYBACK mode and remove the necessity of hitting the actual resources every time we may want to test.
+
+### Playback mode
+
+To run the tests in PLAYBACK mode, set an environment variable called `AZURE_TEST_MODE` and set its value to `PLAYBACK` (If the variable if not set, the default will be `PLAYBACK`). After your variable has been set, change directory to the root folder of the package you're working on and run the `mvn verify` command.
+
+If the tests are successful, we can proceed to run the tests in LIVE mode.
+
+### Live mode
+
+Because in LIVE mode we are hitting an actual resource, we must set the appropriate environment variable to make sure the code tests against the resource we want. Set up an env variable called `COMMUNICATION_LIVETEST_CONNECTION_STRING` and set it to the connection string of the resource you want to test against. 
```

Comment:
`COMMUNICATION_LIVETEST_CONNECTION_STRING` is only used for SMS and Phone Numbers. We use `COMMUNICATION_CONNECTION_STRING` for the rest of the SDKs